### PR TITLE
Removed copy pasta from TCPEndpoint class.

### PR DESCRIPTION
**Pull request by louis89**

### CODE REVIEW

**Comment by asummers** [comment]
Approve.

File: californium-core/src/main/java/org/eclipse/californium/core/network/tcp/CoapClientRegistry.java [hunk 1]
```diff
@@ -16,14 +16,12 @@
 
 /**
  * this class is ment for server tcp connection only
- * 
+ *
  * If you build a TCP server that must serve to multiple TcpClient containing the coapServer (hence multiple coap Client from 1 tcp connection)
- * you must use this registry to keep track of you client.  Once a Connection is establish, the call will be make for you to create a CoAP client 
+ * you must use this registry to keep track of you client.  Once a Connection is establish, the call will be make for you to create a CoAP client
```
Comment (joshlarson) [inline californium-core/src/main/java/org/eclipse/californium/core/network/tcp/CoapClientRegistry.java]: "YOU CLIENT"

Comment (joshlarson) [inline californium-core/src/main/java/org/eclipse/californium/core/network/tcp/CoapClientRegistry.java]: "ONCE A CONNECTION IS ESTABLISH"

Comment (joshlarson) [inline californium-core/src/main/java/org/eclipse/californium/core/network/tcp/CoapClientRegistry.java]: "WILL BE MAKE FOR YOU"

File: californium-core/src/main/java/org/eclipse/californium/core/network/tcp/CoapClientRegistry.java [hunk 2]
```diff
@@ -81,7 +79,7 @@ public final boolean containsCoapClient(final CoapClient client) {
 
 
 	/**
-	 * this will be call when a new connection is established.  
+	 * this will be call when a new connection is established.
```
Comment (joshlarson) [inline californium-core/src/main/java/org/eclipse/californium/core/network/tcp/CoapClientRegistry.java]: "WILL BE CALL"

**Comment by joshlarson** [comment]
APPROVE

**Pull request merged**